### DISCUSSION TITLE
What is the exact `clickhouse_http_handler_port` ?

### DISCUSSION THREAD

**Discussion by hantmac** [databendlabs/databend#5991] [General]
**Summary**
In the documentation, I found that the port is 8124 in some places and 8125 in others, which caused several cases to fail, such as the vector example.
<img width="1028" alt="image" src="https://user-images.githubusercontent.com/7600925/173515974-69982c96-a796-4f30-a50d-bf63237ae65c.png">
<img width="1091" alt="image" src="https://user-images.githubusercontent.com/7600925/173516007-7960a97c-45cd-4f11-aae4-93df32590768.png">

Which specific port is `clickhouse_http_handler_port`?

**Comment by youngsofun** [comment]
default is 8124
in ci, use 8125 for standalone,  8125-8127 for cluster mode

**Comment by hantmac** [comment]
So in this kind of doc https://databend.rs/doc/learn/analyze-nginx-logs-with-databend-and-vector we should remind users to change the clickhouse_http_handler_port.

> default is 8124 in ci, use 8125 for standalone, 8125-8127 for cluster mode
